startups/tests.py

```diff
@@ -1,19 +1,65 @@
+from django.contrib.auth import get_user_model
 from django.urls import reverse
 from rest_framework import status
-from rest_framework.test import APITestCase
+from rest_framework.test import APITestCase, APIClient
 
 from startups.models import StartUpProfile
 
 
-class ProfileStartUpTests(APITestCase):
+class StartUpFilterSearchTests(APITestCase):
 
-    def test_create_sturtup_profile(self):
-        url = reverse('create')
-        data = {'name': 'My first  startup',
-                'description': 'this my first startup project, i want you to give me a lot of money:D',
-                'website': 'www.firststartup.com',
-                }
-        response = self.client.post(url, data, format='json')
-        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
-        self.assertEqual(StartUpProfile.objects.count(), 1)
-        self.assertEqual(StartUpProfile.objects.get().name, 'DabApps')
+    def setUp(self):
+        User = get_user_model()
+        self.user = User.objects.create_user(
+            email="john@gmail.com",
+            password="123456pok",
+            first_name="John",
+            last_name="Doe",
+            user_phone="+1234567890"
+        )
+        self.user.add_role("Investor")
+
+        StartUpProfile.objects.create(user_id=self.user, name="Tech Solutions", description="Innovative tech startup")
+        StartUpProfile.objects.create(user_id=self.user, name="Eco Start", description="Sustainable energy startup")
+        StartUpProfile.objects.create(user_id=self.user, name="Foodie", description="Food delivery service startup")
```

Comment:
You are assigning the user using the field `user_id`. It's recommended to use the related field name directly (e.g., `user=self.user`) for better clarity and readability.

Reply:
I have to rename the user_id field to just user?